### DISCUSSION TITLE
How to set the great clear public profile view and /home layout in version 4.0.x

### DISCUSSION THREAD

**Discussion by jacotec** [mastodon/mastodon#20662] [General]
Dear Mastodon community,

I just did a test run with 4.0.0rc4 on a copy of one of my instances. The new functions sound pretty well, but there are some things which, let's say: I'm completely shocked and I think that's a massive step backwards in clearness and man-machine-interface of the public views.

One of the major pro's for Mastodon is that anyone can have a public visible micro blog home page with his public posts. That's the perfect clear layout in v3.5.3 of a public profile page:

![image](https://user-images.githubusercontent.com/3321655/201699104-b2f61643-2f1b-480e-bcf7-b7e03ce1f258.png)

The same in 4.0.0rc4. 70% of the same area is filled with things which are not interesting for a public visitor. Why are two sidebars narrowing the screen when that information would perfectly fit in one? The complete layout is a mess in my opinion:

![image](https://user-images.githubusercontent.com/3321655/201699671-14839f6f-43f9-4a9b-8674-bdb13d2bd543.png)

So, my question: How can I set the public profile view (for non-logged in visitors) to the same crystal clear layout as in v 3.5.3? Or will I be doomed to stay at 3.5.3 forever?

Second: 

That's the great "/public" page in 3.5.3. Clear layout (the "Finde heraus, was gerade in der Welt los ist" header could be a bit smaller) and a perfect overview about the local posts on this server (if set to not show federated as well). Decent top bar shows that's a Mastodon server and gives all you need to go to the social media functionality.

![image](https://user-images.githubusercontent.com/3321655/201700466-293b9189-8417-4920-8931-e050adcd2ad2.png)

The same in 4.0.0rc4. The left and right sidebars are wasting space which is not of interest for a public visitor who wants to have an overview over public toots, 2/3rd of the screen are nothing but wasted:

![image](https://user-images.githubusercontent.com/3321655/201701273-f144cd7d-da11-4da9-b320-f79857452742.png)

I really apologize if my criticism sounds too harsh under the influence of this shock, and I really, really don't want to bother anyone. But I'm just really shocked about the decrease of  user-friendliness and clearness on the public pages which are absolutely perfect in v3.

I hope there is a chance to alter course before the final release and at least give us admins the chance to choose the style for the public pages and profile views.

**Comment by tapete** [comment]
I agree and would also love to have a more user friendly public view. This would make me use mastodon as a blog and as a fan page for my sports club. 

The next point is that the public view uses the dark theme as default. How did you switched to the light theme?
